example-conf/mesosphere-dcos/mesos-site.xml

```diff
@@ -37,7 +37,7 @@
 
   <property>
     <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
-    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
```

Comment:
Shouldn't we keep 3 masters as the default since production level requires a quorum of 3?